neurodamus/metype.py

```diff
@@ -10,10 +10,23 @@
 from .core.configuration import ConfigurationError, SimConfig
 
 
+class SectionIdError(Exception):
+    pass
+
+
 class BaseCell:
     """Class representing an basic cell, e.g. an artificial cell"""
 
     __slots__ = ("_ccell", "_cellref", "raw_gid")
+    _section_layout = [
+        ("soma", lambda c: c.soma, lambda c: c.nSecSoma),
```

Comment:
I still prefer a simple dataclass so it's obvious what these functions are, it makes it self-documenting

Reply:
But you can leave it this way, I guess.